index.bs

```diff
@@ -72,32 +68,15 @@ beyond those described in the Generic Sensor API [[!GENERIC-SENSOR]].
 Model {#model}
 =====
 
-The Gyroscope's associated <a>Sensor subclass</a>
+The Gyroscope's associated <a>Sensor</a> subclass
 is the {{Gyroscope}} class.
 
-The Gyroscope's associated <a>SensorReading subclass</a>
-is the {{GyroscopeReading}} class.
-
 The Gyroscope has a <a>default sensor</a>,
 which is the device's main gyroscope sensor.
 
-The Gyroscope has a single <a>supported reporting mode</a>
-which is "<a>periodic</a>".
-
-The Gyroscope Sensor's <a>permission</a> name is `"gyroscope"`.
-It has no <a>associated PermissionDescriptor</a>.
-
-The Gyroscope has an associated abstract operation
-to <dfn>retrieve the sensor permission</dfn> which
-must simply return a <a>permission</a> whose name is "gyroscope".
-
-The Gyroscope has an associated abstract operation
-to <dfn lt="Construct SensorReading Object">construct a SensorReading object</dfn>
-which creates a new {{GyroscopeReading}} object and sets each of its
-<a attribute for="GyroscopeReading">x</a>
-<a attribute for="GyroscopeReading">y</a> and
-<a attribute for="GyroscopeReading">z</a> attributes to
-the <a>current angular velocity</a> about the corresponding axis.
+The Gyroscope extends sensor's [=latest reading=] with three [=map/entries=] whose
```

Comment:
It's unclear what "The Gyroscope" refers to, here. Also why the uppercase?

I also would not use the term "extend" here.

There should be a [=latest reading=] per [=sensor=] of type gyroscope.